presto-main/src/main/resources/webapp/src/package.json

```diff
@@ -10,14 +10,14 @@
     "babel-preset-flow": "^6.23.0",
     "babel-preset-react": "^6.24.1",
     "flow-bin": "^0.85.0",
-    "webpack": "^4.16.0",
+    "webpack": "^4.43.0",
```

Comment:
Why did these versions get updated?

Reply:
I'm not sure. It looks like I'm using webpack v4.16.3 to build the js files. I changed the version back to 4.16 in package.json.